src/lib/embeds.ts

```diff
@@ -2,4 +2,5 @@ export enum Color {
   Blue = 0x3498db,
   Green = 0x2ecc71,
   Red = 0xe74c3c,
+  Yellow = 0xe7c83c,
```

Comment:
How about `#f1c40f` to match Discord's default yellow role color? The other colors (Blue, Green, and Red) currently match their respective default role colors.
```suggestion
  Yellow = 0xf1c40f,
```